embassy-usb/src/driver.rs

```diff
@@ -106,6 +100,16 @@ pub trait Bus {
     fn force_reset(&mut self) -> Result<(), Unsupported> {
         Err(Unsupported)
     }
+
+    /// Initiates a remote wakeup of the host by the device.
+    ///
+    /// The default implementation just returns `Unsupported`.
```

Comment:
there's no default impl

Reply:
oops!  😂

Reply:
because `remote_wakeup` is async, a default impl is not possible, so I've just removed the comment.